spec/factories/groups.rb

```diff
@@ -2,8 +2,25 @@
 FactoryGirl.define do
   factory :group do
     size 1
-    status 'full'
     association :leader, factory: :student_in_draw
     draw { leader.draw }
+    members { [leader] }
+
+    factory :full_group do
+      size 2
+      after(:build) do |g|
+        g.draw.suites << create(:suite_with_rooms, rooms_count: g.size)
+        (g.size - g.members.size).times do
+          g.members << create(:student, draw: g.draw)
+        end
+      end
+    end
+
+    factory :open_group do
```

Comment:
yeah I think we leave this for now; I guess we could throw an exception or something but it seems like overkill